go/examples/directory-receptor/generators/list_all.go

```diff
@@ -0,0 +1,38 @@
+package generators
+
+import (
+	"github.com/trustero/monorepo/receptor-sdk/client/api/model"
+	"os/exec"
+)
+
+type ListAll struct{}
+
+func (m ListAll) Verify(credentials string) (successful bool, err error) {
+	successful = true
+	return
+}
+
+func (m ListAll) ReceptorModelId() string {
+	return "trr-100000"
+}
+
+func (m ListAll) ServiceModelId() string {
+	return "trs-100000"
+}
```

Comment:
i don't understand why this isn't returning a list of services.
i think we also said that we would change this to some "name"

Reply:
Okay, I think I understand now why it's not a LIST of services -- this file is generating only a single piece of Evidence, and we decided that we would not have aggregated evidence any longer, only evidence per "platform+service".
So, some example captions we might see in the product:

- AWS IAM Users list
- Azure IAM Users list
- AWS S3 Asset inventory

I guess my next question is: what happens for Recruitee, where the platform IS the service?
- Recruitee Recruitee Job Descriptions

What about Github, where "Dependabot" is currently treated as a service?
- Github Dependabot configuration
- Github ?Repository? configuration
Or maybe dependabot should not be a separate service.